paper-collapse-group.html

```diff
@@ -89,7 +89,7 @@
 			this._items.forEach(function(item) {
 				// Force all other items (aside the one triggering the event) to
 				// get closed
-				if (item !== e.target) {
+				if (item !== e.target && true === e.target.opened) {
```

Comment:
Could you please flip the operands => `e.target.opened === true` (for code consistency)?

Reply:
Sure , I will do it later this day :)